README.rst

```diff
@@ -173,6 +173,11 @@ The injector provides itself as the `'injector'` service::
     with Injector() as injector:
         injector.get('injector')
 
+Local values can be provided when constructing the injector::
+
+    with RequestInjector(request=request) as injector:
+        injector.get('form:username')
+
```

Comment:
These changes should go into README.rst.in given the project's custom doc builder.